lib/msal-browser/src/app/Configuration.ts

```diff
@@ -63,7 +63,6 @@ export type SystemOptions = {
  *
  */
 export type FrameworkOptions = {
-    isAngular?: boolean;
     unprotectedResources?: Array<string>;
     protectedResourceMap?: Map<string, Array<string>>;
 };
```

Comment:
How are we handling "Angular" broadcast messages then?

Reply:
This will be handled within the angular library. The common library doesn't need to know about angular broadcasts.

Reply:
Rather than implementing the Angular code in common, we should emit events that wrapper libraries can hook into.